odc_eks/waf.tf

```diff
@@ -49,12 +147,13 @@ module "waf_label" {
   delimiter  = "-"
 }
 
-# The module which is defined on repository: https://github.com/traveloka/terraform-aws-waf-owasp-top-10-rules
-# For a better understanding of what are those parameters mean,
-# please read the description of each variable in the variables.tf file:
-# https://github.com/traveloka/terraform-aws-waf-owasp-top-10-rules/blob/master/variables.tf
+# Module: https://github.com/masterpointio/terraform-aws-waf-owasp-top-10-rules
+# The module is a fork from repository: https://github.com/traveloka/terraform-aws-waf-owasp-top-10-rules
+# PR raised to support below features: https://github.com/traveloka/terraform-aws-waf-owasp-top-10-rules/pull/17
+# - Updates to address all Terraform 0.12 warnings
+# - Updates to allow disable specific XSS and  rules
 module "owasp_top_10_rules" {
-  source = "git::https://github.com/traveloka/terraform-aws-waf-owasp-top-10-rules.git?ref=v0.2.0"
+  source = "git::https://github.com/masterpointio/terraform-aws-waf-owasp-top-10-rules.git?ref=master"
```

Comment:
Please change to point to ODC fork.

Reply:
changes DONE.
- created a fork https://github.com/opendatacube/terraform-aws-waf-owasp-top-10-rules and updated source to point to ODC fork